include/swift/AST/DiagnosticsSIL.def

```diff
@@ -965,7 +965,7 @@ ERROR(regionbasedisolation_transfer_yields_race_stronglytransferred_binding, non
       "value of non-Sendable type %0 accessed after being transferred; later accesses could race",
       (Type))
 ERROR(regionbasedisolation_arg_transferred, none,
-      "%0 value of type %1 transferred to %2 context; later accesses to value could race",
+      "sending %0 value of type %1 to %2 context; later accesses to value could race",
```

Comment:
What's the difference between this message and the previous two that were changed? It seems the same with slightly different wording. In any case, my suggestion applied to this message would be something like

`sending %0 value of type %1 to %2 context with later accesses risks a data race `

Reply:
I left this in b/c I haven't renamed transferring -> sendable.

Reply:
Actually this is a fallback.